include/ignition/rendering/BoundingBoxCamera.hh

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef IGNITION_RENDERING_BoundingBoxCAMERA_HH_
+#define IGNITION_RENDERING_BoundingBoxCAMERA_HH_
+
+#include <string>
+#include <vector>
+#include <cstdint>
+
+#include <ignition/common/Event.hh>
+#include "ignition/rendering/Camera.hh"
+
+
+namespace ignition
+{
+  namespace rendering
+  {
+    /// \brief BoundingBox types for Visible / Full BoundingBox
+    enum class BoundingBoxType {
+
+      /// FullBox2D: 2D box that shows the full box of occluded objects
```

Comment:
It will not be readable here if it becomes uppercase :D 
I see that notation is also used .. https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo5/include/ignition/gazebo/Types.hh#L63

can we keep that also :D

Reply:
let's make these uppercase for consistency within ign-rendering. It should also have a prefix, i.e. `BBT_FULLBOX2D`